src/I18n/Formatter/IcuFormatter.php

```diff
@@ -51,7 +51,7 @@ public function format($locale, $message, array $vars)
             return $this->_formatMessage($locale, $message, $vars);
         }
 
-        if (isset($vars['_context']) && isset($message['_context'])) {
+        if (isset($vars['_context'], $message['_context'])) {
```

Comment:
Not sure if that is better, because not everyone knows if that means `&&` or `||`.
So I am -1 on those changes.